Build failure: sirikali

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

I tried to update and rebuild my system
```
• Updated input 'home-manager':
    'github:nix-community/home-manager/72526a5f7cde2ef9075637802a1e2a8d2d658f70?narHash=sha256-IJksPW32V9gid9vDxoloJMRk%2BYGjxq5drFHBFeBkKU8%3D' (2025-04-17)
  → 'github:nix-community/home-manager/5e6a8203cee7cc33b2e0d9a0adb7268f46447292?narHash=sha256-IVioWVz5qVtHiqosesW7CJW//m/yADr7cVdgF1P4N8s%3D' (2025-04-18)
• Updated input 'nixpkgs':
    'github:nixos/nixpkgs/2631b0b7abcea6e640ce31cd78ea58910d31e650?narHash=sha256-LWqduOgLHCFxiTNYi3Uj5Lgz0SR%2BXhw3kr/3Xd0GPTM%3D' (2025-04-12)
  → 'github:nixos/nixpkgs/b024ced1aac25639f8ca8fdfc2f8c4fbd66c48ef?narHash=sha256-fusHbZCyv126cyArUwwKrLdCkgVAIaa/fQJYFlCEqiU%3D' (2025-04-17)
building the system configuration...
```

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/295121262

### Relevant log output

```console
Running phase: qtPreHook
@nix { "action": "setPhase", "phase": "qtPreHook" }
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/v178cblv34acyana31pplkpqyhxlfpcr-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMA>
-- The C compiler identification is GNU 14.2.1
-- The CXX compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/f0m6caffiykyvsjim9376a3hx2yj2ghj-gcc-wrapper-14.2.1.20250322/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE
-- Found OpenGL: /nix/store/sq50878z0rdxwdyjw3fvbgf6wkpcnpcp-libglvnd-1.7.0/lib/libOpenGL.so
-- Found WrapOpenGL: TRUE
-- Found WrapVulkanHeaders: /nix/store/xwhkngnns6ax8fgl7igdd6pqwkav9n5b-vulkan-headers-1.4.309.0/include
-- Setting C++ version to C++14
-- Found PkgConfig: /nix/store/bbp56ai8w43xkfisar83h37imjdlcx6r-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Checking for module 'pwquality'
--   Found pwquality, version 1.4.5
-- found gcrypt header file at: /nix/store/llhp6iins20c1899z6v4zsxd16jjrhwm-libgcrypt-1.10.3-dev/include/gcrypt.h
-- found gcrypt library at: /nix/store/3m732af9shqzjzxh1ams7clfcdkmwvbp-libgcrypt-1.10.3-lib/lib/libgcrypt.so
-- adding library search path of: /nix/store/3m732af9shqzjzxh1ams7clfcdkmwvbp-libgcrypt-1.10.3-lib/lib
-- Found Qt6Widgets, version  6.9.0
-- Found Qt6Core, version  6.9.0
-- Found Qt6Network, version  6.9.0
-- Found Qt6DBus, version  6.9.0
-- ---------------------------------------------------------------------------
-- mhogomchungu_task library will be provided by an internal library
-- ---------------------------------------------------------------------------
-- Checking for module 'lxqt-wallet'
--   No package 'lxqt-wallet' found
-- ERROR: could not find lxqt_wallet package
-- ---------------------------------------------------------------------------
-- lxqt wallet support will be provided by an internal library
-- ---------------------------------------------------------------------------
-- Found gcrypt header file: /nix/store/llhp6iins20c1899z6v4zsxd16jjrhwm-libgcrypt-1.10.3-dev/include/gcrypt.h
-- Found gcrypt library: /nix/store/3m732af9shqzjzxh1ams7clfcdkmwvbp-libgcrypt-1.10.3-lib/lib/libgcrypt.so
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Checking for modules 'KF6Wallet'
--   Found KF6Wallet, version 6.13.0
-- 
--------------------------------------------------------------------------
-- kwallet support found, will build kwallet functionality
-- -----------------------------------------------------------------------
-- 
--------------------------------------------------------------------------
-- libsecret support found, will build libsecret functionality
-- -----------------------------------------------------------------------
-- ---------------------------------------------------------------------------
-- Building using Qt6
-- ---------------------------------------------------------------------------


-- Configuring done (11.5s)
-- Generating done (0.4s)
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025
    KDE_INSTALL_APPDIR
    KDE_INSTALL_AUTOSTARTDIR
    KDE_INSTALL_BINDIR
    KDE_INSTALL_CMAKEPACKAGEDIR
    KDE_INSTALL_CONFDIR
    KDE_INSTALL_DATADIR
    KDE_INSTALL_DATAROOTDIR
    KDE_INSTALL_DBUSDIR
    KDE_INSTALL_DBUSINTERFACEDIR
    KDE_INSTALL_DBUSSERVICEDIR
    KDE_INSTALL_DBUSSYSTEMSERVICEDIR
    KDE_INSTALL_DESKTOPDIR
    KDE_INSTALL_DOCBUNDLEDIR
    KDE_INSTALL_EXECROOTDIR
    KDE_INSTALL_ICONDIR
    KDE_INSTALL_INCLUDEDIR
    KDE_INSTALL_INFODIR
    KDE_INSTALL_KAPPTEMPLATESDIR
    KDE_INSTALL_KCFGDIR
    KDE_INSTALL_KCONFUPDATEDIR
    KDE_INSTALL_KFILETEMPLATESDIR
    KDE_INSTALL_KNOTIFYRCDIR
    KDE_INSTALL_KXMLGUIDIR
    KDE_INSTALL_LIBDIR
    KDE_INSTALL_LIBEXECDIR
    KDE_INSTALL_LOCALEDIR
    KDE_INSTALL_LOCALSTATEDIR
    KDE_INSTALL_LOGGINGCATEGORIESDIR
    KDE_INSTALL_MANDIR
    KDE_INSTALL_METAINFODIR
    KDE_INSTALL_MIMEDIR
    KDE_INSTALL_PLUGINDIR
    KDE_INSTALL_QCHDIR
    KDE_INSTALL_QMLDIR
    KDE_INSTALL_QTPLUGINDIR
    KDE_INSTALL_QTQCHDIR
    KDE_INSTALL_SBINDIR
    KDE_INSTALL_SHAREDSTATEDIR
    KDE_INSTALL_SOUNDDIR
    KDE_INSTALL_SYSCONFDIR
    KDE_INSTALL_SYSTEMDUNITDIR
    KDE_INSTALL_SYSTEMDUSERUNITDIR
    KDE_INSTALL_TEMPLATEDIR
    KDE_INSTALL_WALLPAPERDIR


-- Build files have been written to: /build/source/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: -j4 SHELL=/nix/store/9nw8b61s8lfdn8fkabxhbz0s775gjhbr-bash-5.2p37/bin/bash
[  1%] Building C object src/3rdParty/lxqt_wallet/frontend/libsecret/CMakeFiles/secretlib.dir/__/libsecret.c.o
[  2%] Building C object src/3rdParty/lxqt_wallet/backend/CMakeFiles/lxqtwallet-backend.dir/lxqtwallet.c.o
[  2%] Built target mhogomchungu_task_autogen_timestamp_deps
[  3%] Automatic MOC and UIC for target mhogomchungu_task
[  3%] Built target mhogomchungu_task_autogen
[  5%] Building CXX object src/3rdParty/tasks/CMakeFiles/mhogomchungu_task.dir/mhogomchungu_task_autogen/mocs_compilation.cpp.o
[  6%] Linking C static library libsecretlib.a
[  6%] Built target secretlib
[  7%] Linking C static library liblxqtwallet-backend.a
[  7%] Built target lxqtwallet-backend
[  7%] Built target lxqt-wallet_autogen_timestamp_deps
[  9%] Automatic MOC and UIC for target lxqt-wallet
AutoMoc: /build/source/src/3rdParty/lxqt_wallet/frontend/lxqt_wallet.h: note: No relevant classes found. No output generated.
[  9%] Built target lxqt-wallet_autogen
[ 10%] Building CXX object src/3rdParty/lxqt_wallet/frontend/CMakeFiles/lxqt-wallet.dir/lxqt-wallet_autogen/mocs_compilation.cpp.o
[ 11%] Building CXX object src/3rdParty/lxqt_wallet/frontend/CMakeFiles/lxqt-wallet.dir/lxqt_kwallet.cpp.o
[ 12%] Building CXX object src/3rdParty/lxqt_wallet/frontend/CMakeFiles/lxqt-wallet.dir/lxqt_libsecret.cpp.o
[ 14%] Linking CXX static library libmhogomchungu_task.a
[ 14%] Built target mhogomchungu_task
[ 15%] Building CXX object src/3rdParty/lxqt_wallet/frontend/CMakeFiles/lxqt-wallet.dir/changepassworddialog.cpp.o
[ 16%] Building CXX object src/3rdParty/lxqt_wallet/frontend/CMakeFiles/lxqt-wallet.dir/password_dialog.cpp.o
[ 18%] Building CXX object src/3rdParty/lxqt_wallet/frontend/CMakeFiles/lxqt-wallet.dir/lxqt_windows_dpapi.cpp.o
[ 19%] Building CXX object src/3rdParty/lxqt_wallet/frontend/CMakeFiles/lxqt-wallet.dir/lxqt_internal_wallet.cpp.o
[ 20%] Building CXX object src/3rdParty/lxqt_wallet/frontend/CMakeFiles/lxqt-wallet.dir/lxqt_wallet.cpp.o
[ 20%] Built target sirikali.pkexec_autogen_timestamp_deps
[ 22%] Building CXX object src/3rdParty/lxqt_wallet/frontend/CMakeFiles/lxqt-wallet.dir/lxqt_osx_keychain.cpp.o
[ 23%] Automatic MOC and UIC for target sirikali.pkexec
[ 23%] Built target sirikali.pkexec_autogen
[ 24%] Building CXX object src/siripolkit/CMakeFiles/sirikali.pkexec.dir/sirikali.pkexec_autogen/mocs_compilation.cpp.o
[ 25%] Building CXX object src/siripolkit/CMakeFiles/sirikali.pkexec.dir/main.cpp.o
[ 27%] Building CXX object src/siripolkit/CMakeFiles/sirikali.pkexec.dir/zulupolkit.cpp.o
[ 28%] Linking CXX static library liblxqt-wallet.a
[ 28%] Built target lxqt-wallet
[ 28%] Built target sirikali_autogen_timestamp_deps
[ 29%] Automatic MOC and UIC for target sirikali
[ 31%] Linking CXX executable sirikali.pkexec
[ 31%] Built target sirikali.pkexec
[ 31%] Built target sirikali_autogen
[ 32%] Automatic RCC for src/icon.qrc
[ 33%] Building CXX object CMakeFiles/sirikali.dir/sirikali_autogen/mocs_compilation.cpp.o
[ 35%] Building CXX object CMakeFiles/sirikali.dir/src/dialogok.cpp.o
[ 36%] Building CXX object CMakeFiles/sirikali.dir/src/main.cpp.o
[ 37%] Building CXX object CMakeFiles/sirikali.dir/src/systemsignalhandler.cpp.o
[ 38%] Building CXX object CMakeFiles/sirikali.dir/src/crypto.cpp.o
[ 40%] Building CXX object CMakeFiles/sirikali.dir/src/debugwindow.cpp.o
[ 41%] Building CXX object CMakeFiles/sirikali.dir/src/configoptions.cpp.o
[ 42%] Building CXX object CMakeFiles/sirikali.dir/src/configfileoption.cpp.o
[ 44%] Building CXX object CMakeFiles/sirikali.dir/src/sirikali.cpp.o
[ 45%] Building CXX object CMakeFiles/sirikali.dir/src/checkforupdateswindow.cpp.o
[ 46%] Building CXX object CMakeFiles/sirikali.dir/src/keydialog.cpp.o
[ 48%] Building CXX object CMakeFiles/sirikali.dir/src/siritask.cpp.o
[ 49%] Building CXX object CMakeFiles/sirikali.dir/src/help.cpp.o
/build/source/src/keydialog.cpp: In member function 'void keyDialog::setUpInitUI()':
/build/source/src/keydialog.cpp:229:57: warning: 'void QCheckBox::stateChanged(int)' is deprecated: Use checkStateChanged() instead [-Wdeprecated-declarations]
  229 |         connect( m_ui->checkBoxOpenReadOnly,&QCheckBox::stateChanged,
      |                                                         ^~~~~~~~~~~~
In file included from /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtWidgets/QCheckBox:1,
                 from /build/source/build/sirikali_autogen/include/ui_keydialog.h:14,
                 from /build/source/src/keydialog.cpp:21:
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtWidgets/qcheckbox.h:41:10: note: declared here
   41 |     void stateChanged(int);
      |          ^~~~~~~~~~~~
[ 50%] Building CXX object CMakeFiles/sirikali.dir/src/themes.cpp.o
[ 51%] Building CXX object CMakeFiles/sirikali.dir/src/createbackendwindow.cpp.o
[ 53%] Building CXX object CMakeFiles/sirikali.dir/src/engines.cpp.o
[ 54%] Building CXX object CMakeFiles/sirikali.dir/src/oneinstance.cpp.o
[ 55%] Building CXX object CMakeFiles/sirikali.dir/src/mountinfo.cpp.o
[ 57%] Building CXX object CMakeFiles/sirikali.dir/src/utility.cpp.o
[ 58%] Building CXX object CMakeFiles/sirikali.dir/src/dialogmsg.cpp.o
[ 59%] Building CXX object CMakeFiles/sirikali.dir/src/favorites.cpp.o
[ 61%] Building CXX object CMakeFiles/sirikali.dir/src/favorites2.cpp.o
[ 62%] Building CXX object CMakeFiles/sirikali.dir/src/checkforupdates.cpp.o
[ 63%] Building CXX object CMakeFiles/sirikali.dir/src/plugin.cpp.o
/build/source/src/favorites2.cpp: In constructor 'favorites2::favorites2(QWidget*, secrets&, std::function<void()>, const engines::engine&, const QString&)':
/build/source/src/favorites2.cpp:180:60: warning: 'void QCheckBox::stateChanged(int)' is deprecated: Use checkStateChanged() instead [-Wdeprecated-declarations]
  180 |         connect( m_ui->cbSetMountPathNotPrefix,&QCheckBox::stateChanged,[ this ]( int s ){
      |                                                            ^~~~~~~~~~~~
In file included from /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtWidgets/QCheckBox:1,
                 from /build/source/build/sirikali_autogen/include/ui_favorites2.h:14,
                 from /build/source/src/favorites2.cpp:21:
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtWidgets/qcheckbox.h:41:10: note: declared here
   41 |     void stateChanged(int);
      |          ^~~~~~~~~~~~
/build/source/src/favorites2.cpp:579:51: warning: 'void QCheckBox::stateChanged(int)' is deprecated: Use checkStateChanged() instead [-Wdeprecated-declarations]
  579 |         connect( m_ui->cbShowPassword,&QCheckBox::stateChanged,[ this ]( int s ){
      |                                                   ^~~~~~~~~~~~
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtWidgets/qcheckbox.h:41:10: note: declared here
   41 |     void stateChanged(int);
      |          ^~~~~~~~~~~~
[ 64%] Building CXX object CMakeFiles/sirikali.dir/src/tablewidget.cpp.o
[ 66%] Building CXX object CMakeFiles/sirikali.dir/src/readonlywarning.cpp.o
[ 67%] Building CXX object CMakeFiles/sirikali.dir/src/secrets.cpp.o
[ 68%] Building CXX object CMakeFiles/sirikali.dir/src/filemanager.cpp.o
[ 70%] Building CXX object CMakeFiles/sirikali.dir/src/utility2.cpp.o
[ 71%] Building CXX object CMakeFiles/sirikali.dir/src/win.cpp.o
[ 72%] Building CXX object CMakeFiles/sirikali.dir/src/processManager.cpp.o
[ 74%] Building CXX object CMakeFiles/sirikali.dir/src/settings.cpp.o
[ 75%] Building CXX object CMakeFiles/sirikali.dir/src/engines/options.cpp.o
[ 76%] Building CXX object CMakeFiles/sirikali.dir/src/engines/cryfs.cpp.o
[ 77%] Building CXX object CMakeFiles/sirikali.dir/src/engines/ecryptfs.cpp.o
[ 79%] Building CXX object CMakeFiles/sirikali.dir/src/engines/gocryptfs.cpp.o
/build/source/src/engines/options.cpp: In constructor 'options::options(const engines::engine&, const engines::engine::mountGUIOptions&)':
/build/source/src/engines/options.cpp:55:45: warning: 'void QCheckBox::stateChanged(int)' is deprecated: Use checkStateChanged() instead [-Wdeprecated-declarations]
   55 |         connect( m_ui->checkBox,&QCheckBox::stateChanged,[ this ]( bool e ){
      |                                             ^~~~~~~~~~~~
In file included from /nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtWidgets/QCheckBox:1,
                 from /build/source/src/engines/options.h:33,
                 from /build/source/src/engines/options.cpp:20:
/nix/store/cp30fin83aimfljxb5988bkn2micgi04-qtbase-6.9.0/include/QtWidgets/qcheckbox.h:41:10: note: declared here
   41 |     void stateChanged(int);
      |          ^~~~~~~~~~~~
[ 80%] Building CXX object CMakeFiles/sirikali.dir/src/engines/securefs.cpp.o
[ 81%] Building CXX object CMakeFiles/sirikali.dir/src/engines/sshfs.cpp.o
[ 83%] Building CXX object CMakeFiles/sirikali.dir/src/engines/customcreateoptions.cpp.o
/build/source/src/engines/securefs.cpp: In member function 'QByteArray securefs::extraLogOutput(const engines::engine::args&) const':
/build/source/src/engines/securefs.cpp:263:56: error: invalid initialization of reference of type 'QFile&&' from expression of type 'const QString'
  263 |                                         return _get_log( opts[ s + 1 ] ) ;
      |                                                ~~~~~~~~^~~~~~~~~~~~~~~~~
/build/source/src/engines/securefs.cpp:220:37: note: in passing argument 1 of 'QByteArray _get_log(QFile&&)'
  220 | static QByteArray _get_log( QFile&& file )
      |                             ~~~~~~~~^~~~
make[2]: *** [CMakeFiles/sirikali.dir/build.make:601: CMakeFiles/sirikali.dir/src/engines/securefs.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:267: CMakeFiles/sirikali.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
lines 231-272/272 (END)
```

### Additional context

If you check, hydra builds for many other platforms fail:

https://hydra.nixos.org/search?query=sirikali

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.23, NixOS, 25.05 (Warbler), 25.05.20250412.2631b0b`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.1`
 - nixpkgs: `/nix/store/5ds20jm3x2s4z7wn3581r6lc9ybmh45b-source`

### Notify maintainers

@linuxissuper 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/si/sirikali/package.nix
+++ b/pkgs/by-name/si/sirikali/package.nix
@@ -22,13 +22,13 @@
 
 stdenv.mkDerivation rec {
   pname = "sirikali";
-  version = "1.8.1";
+  version = "1.8.2";
 
   src = fetchFromGitHub {
     owner = "mhogomchungu";
     repo = "sirikali";
     rev = version;
-    hash = "sha256-YGMmDatTKjhORTisX9kTVrpIWo6j0YkxZsDLJXga6ho=";
+    hash = "sha256-rfmWtbPYtkaGemeStMWwA6JllOkDiHMftSfmirtAOEQ=";
   };
 
   buildInputs =

